github-templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+Status: **Ready for Review** or **Open for Visibility**
+Reviewers: GitHub @<name>s of all reviewers
+
+JIRA: <link to applicable JIRA ticket>
+Linked PRs: <links to corresponding PRs, optional>
+
+## Changes
+- <changes here>
+
+## Todos:
+- [ ] +1
+- [ ] Passes tests
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] <Other TODOs>
```

Comment:
Should we treat this as a meta-template?  Maybe `[ ] <Other TODOs applicable to your team/project>` ?